workers/ohsome_quality_analyst/oqt.py

```diff
@@ -27,12 +27,15 @@ async def create_indicator(
 
     An indicator can be created in two ways:
 
-    One; Calculate from scratch for an area of interest.
-    This is done by providing a bounding polygon as input parameter.
+    1. From Scratch: Calculate from scratch for an area of interest.
+    This is done by providing a GeoJSON object as input parameter.
```

Comment:
I think `GeoJSON object` isn't technically correct, as this could be e.g. a Geometry as well. I would write something like `GeoJSON Feature` or `GeoJSON Feature or FeatureCollection` (if/when that's implemented)

Reply:
Good catch. In fact only GeoJSON Feature is a valid input here.

GeoJSON Feature/FeatureCollection or Polygon/MultiPolygon is or will be a valid input to CLI and API. But oqt.py expects a Feature.

I updated the docstring.